packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -427,6 +427,9 @@ const BuySellForm = props => {
                                                             initial_character_count={buy_sell_store.payment_info.length}
                                                             max_characters={300}
                                                             disabled={should_disable_field}
+                                                            onBlur={event => {
```

Comment:
Do we need to check if event.target is not nullish? Sometimes I had this issue for a dropdown component, for example when clicking outside of the dropdown component to close it